src/common/utils/registry.h

```diff
@@ -378,6 +378,12 @@ namespace registry
                     std::wstring kindMapPath = L"Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\KindMap";
                     changes.push_back({ HKEY_LOCAL_MACHINE, kindMapPath, fileType, fileKindType, false});
                 }
+                if (fileType == L".reg")
+                {
+                    // this regfile registry key has precedence over Software\Classes\.reg for .reg files
+                    std::wstring regfilePath = L"Software\\Classes\\regfile\\shellex\\{8895b1c6-b41f-4c1c-a562-0d564250836f}\\";
```

Comment:
let's make a _CLSID constant for it as well

Reply:
done